Aktas2012/Aktas2012.md

```diff
@@ -0,0 +1,16 @@
+# Aktas 2004
+
+Can B. Aktas. Ph.D. 2012. Impact of lifetime on US residential building LCA results. The International Journal of Life Cycle Assessment https://link.springer.com/article/10.1007%2Fs11367-011-0363-x
+
+## Figure 5
+
+- Image source: Image from the publisher
+- Tool: [WebPlotDigitizer](https://apps.automeris.io/wpd/) v4.1 (WPD)
+- Dataset contains:
+  - WPD JSON project file (folder [WPD_Fig5](WPD_Fig5))
+  - CSV exported from WPD with manual corrections (file [Fig5.csv](Fig5.csv))
+    - first column : American Housing Survey Year
```

Comment:
Please follow the description as suggested in Chastas eta l. 2016. Also applies to the other publications